docs/conf.py

```diff
@@ -124,7 +129,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
Yes, that's why. FWIW I'm currently updating the tox.ini to support the coverage support in the other Pylons repos (100% on py2 + py3). That'll be a separate PR.